src/shared/common/forms/__tests__/dropdown.spec.js

```diff
@@ -64,5 +87,28 @@ describe('Form dropdown component', () => {
         expect(mockCallback).toHaveBeenCalledTimes(2);
         expect(mockCallback).toHaveBeenCalledWith({ 'dropdown': secondValue });
     });
-});
 
+    test('should render only active options (one inactive dropdown value), default is active', () => {
+        const wrapper = render(<Dropdown name="radio-group" conditionChoices={conditionChoices}
+            data={{ 'conditionValue': 'b' }} updateState={() => null} />);
+
+        expect(wrapper).toBeDefined();
+
+        expect(screen.getByText('isA')).toBeInTheDocument();
+        expect(screen.getByText('isB')).toBeInTheDocument();
+        expect(screen.queryByText('isC')).not.toBeInTheDocument();
+    });
+
+    test('should render only active options and dont fetch inactive ones (multiple dropdown values), default is active', () => {
+        const wrapper = render(<Dropdown name="radio-group" conditionChoices={conditionChoices}
+            data={{ 'conditionValue': 'a' }} updateState={() => null} />);
+
+        expect(wrapper).toBeDefined();
+
+        expect(screen.getByText('isA')).toBeInTheDocument();
+        expect(screen.getByText('isB')).toBeInTheDocument();
+        expect(screen.queryByText('isC')).not.toBeInTheDocument();
+        expect(screen.queryByText('isD')).not.toBeInTheDocument();
+    });
+
+});
```

Comment:
We only need one of these tests - it should confirm that the two active from the chosen option are showing, and the inactive current choice, and the three from the other choice are not showing.

The text will need to be updated to match the label changes in the comment above.